src/library/simplified/extension/price-per-person.js

```diff
@@ -14,6 +14,10 @@ else if((utag.isHCO()|| utag.isFCO() || utag.isPCO()|| utag.isLXCO() || utag.isM
         b['pricePerPerson'] = (b['checkout.cartTotal.amount'] / b['numberOfGuests']).toFixed(2);
     }
 }
+else if (utag.isFRateDetails() && (b['entity.flight.flight.avgPriceOfAdultAndSenior.amount'] || b['entity.flight.flight.avgPriceOfChildrenAndInfants.amount']))
+{
+    b['pricePerPerson'] = b['entity.flight.flight.avgPriceOfAdultAndSenior.amount'] + "|" + b['entity.flight.flight.avgPriceOfChildrenAndInfants.amount'];
```

Comment:
I'm not seeing entity.flight.flight.avgPriceOfAdultAndSenior.amount and entity.flight.flight.avgPriceOfChildrenAndInfants.amount on the Flight UDP page when I searched; therefore, pricePerPerson is empty in my case. I do see entity.tripDetails.flightOffer.pricePerTraveler.amount with the price. Should we also check for this field?